Orange/tests/test_distances.py

```diff
@@ -436,148 +442,155 @@ def test_spearmanr_distance_one_example(self):
-                                                 [ 0.25,  0.25,  0.25,  0.25,  0.25,  0.25,  0.  ,  0.25,  0.  ],
-                                                 [ 0.25,  0.  ,  0.25,  0.  ,  0.  ,  0.  ,  0.25,  0.  ,  0.25],
-                                                 [ 0.25,  0.25,  0.25,  0.25,  0.25,  0.25,  0.  ,  0.25,  0.  ]]))
+                                       np.array([[0., 0.25, 0., 0.25, 0.25, 0.25, 0.25, 0.25, 0.25],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0., 0.25, 0., 0.25, 0.25, 0.25, 0.25, 0.25, 0.25],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0., 0.25, 0.],
+                                                 [0.25, 0., 0.25, 0., 0., 0., 0.25, 0., 0.25],
+                                                 [0.25, 0.25, 0.25, 0.25, 0.25, 0.25, 0., 0.25, 0.]]))
         np.testing.assert_almost_equal(self.dist(self.breast[:3], self.breast[:4]),
-                                       np.array([[ 0.        ,  0.49166667,  0.075     ,  0.38333333],
-                                                 [ 0.49166667,  0.        ,  0.38333333,  0.46666667],
-                                                 [ 0.075     ,  0.38333333,  0.        ,  0.36666667]]))
+                                       np.array([[0., 0.49166667, 0.075, 0.38333333],
+                                                 [0.49166667, 0., 0.38333333, 0.46666667],
+                                                 [0.075, 0.38333333, 0., 0.36666667]]))
         np.testing.assert_almost_equal(self.dist(self.breast[3], self.breast[:4]),
-                                       np.array([[ 0.3833333,  0.4666667,  0.3666667,  0.       ]]))
+                                       np.array([[0.3833333, 0.4666667, 0.3666667, 0.]]))
         np.testing.assert_almost_equal(self.dist(self.breast[:4], self.breast[3]),
-                                       np.array([[ 0.3833333],
-                                                 [ 0.4666667],
-                                                 [ 0.3666667],
-                                                 [ 0.       ]]))
+                                       np.array([[0.3833333],
+                                                 [0.4666667],
+                                                 [0.3666667],
+                                                 [0.]]))
         np.testing.assert_almost_equal(self.dist(self.breast[:3], self.breast[:3], axis=0),
-                                       np.array([[ 0.  ,  0.25,  0.  ,  0.25,  0.25,  0.25,  0.25,  0.25,  0.25],
```

Comment:
@kernc With arrays like this, I believe that having the previous formatting is better (readability is usually more important). The case here is, in the old style with spaces, it's easy to see that the diagonal is all zeros, in the new style it's not.

As for pylint, all these tables could be moved to a file that would just contain them as constants with good names. and then in that file we could ignore multiple spaces in arrays and even lines longer than 80.